_posts/2018-11-20-mongodb-europe-018.md

```diff
@@ -0,0 +1,411 @@
+
+
+# Table of contents
+1. [Atlas](#atlas)
+2. [MongoDB University](#mongodb-university)
+3. [Compass Aggregation Pipeline Builder](#compass-aggregation-pipeline-builder)
+4. [Common query mistakes](#common-query-mistakes)
+5. [Stitch](#stitch)
+6. [Meet the experts](#meet-the-experts)
+7. [Streaming data pipelines with MongoDB and Kafka at AO](#streaming-data-pipelines-with-mongodb-and-kafka-at-ao)
+8. [MongoDB Enterprise Operator for Kubernetes at Amadeus](#mongodb-enterprise-operator-for-kubernetes-at-amadeus)
+9. [MongoDB Charts](#mongodb-charts)
+10. [Closing](#closing)
+
+# Atlas
+([MongoDB Atlas for your Enterprise](https://sched.co/FmAF){:target="_blank" rel="noopener noreferrer"}, Vladislava Stevanovic & Veronica Tudor)
+
+With Atlas, MongoDB brings us DBaaS (Database As A Service).
+You can run your database in the cloud with the cloud provider of your choice.
+Multiple options to secure your database are built in by default.
+Backups are taken automatically.
+
+### Getting started
+It is very easy to get started since a free tier is provided for everyone and you can deploy with the cloud provider of your choice (Azure , GCP, AWS).
+A cluster can be started for free in just a few clicks.
+
+<div style="text-align: center;">
+  <img src="/img/2018-11-08-mongodb-europe-2018/create-mongodb-atlas-cluster.png" width="100%">
+</div>
+
+Start your own cluster: [https://cloud.mongodb.com](https://cloud.mongodb.com){:target="_blank" rel="noopener noreferrer"}
+
+### Cloud
+MongoDB Atlas is a cross-platform database. 
+You can run it on Amazon Web Services, Google Cloud or Microsoft Azure. 
+It provides you an availability map that shows you which users are served from where an what latency should be expected.
+<div style="text-align: center;">
+  <img src="/img/2018-11-08-mongodb-europe-2018/global-clusters.png" width="100%">
+</div>
+
+### Scalability
+When your database grows it is easy to scale up or out.
+You can scale up by increasing the size of the instance on which your database runs.
+Scaling out is done by the process of sharding.
+Here we are storing data of the same collection across multiple smaller partitions so we can distribute these partitions over multiple machines and increase read-write performance.
+This way you do not run into the limitations of a single server.
+
+To ensure that MongoDB stores data equally across shards you need the right strategy of choosing a partition key.
+
+### High availability 
+When a primary node goes down, a new primary is chosen immediately by the system of voting.
+All nodes vote on who should become the new primary. 
+The node with the majority of the votes becomes the new primary.
+A general guideline is to have a replica set that consists of one primary node and at least two secondary nodes.
+To ensure maximum uptime the procedure to recover from instance failures is completely automated.
+
+
+# MongoDB University
+(General Session Keynote)
```

Comment:
Why is this here?

Reply:
because it was in the Keynote but can be left out, it looks different from the other titles